manifests/a/alibaba/dingtalk/6.0.15-Release.5171263/alibaba.dingtalk.yaml

```
@@ -1,15 +1,15 @@
 PackageIdentifier: "alibaba.dingtalk"
-PackageVersion: "6.0.15.5171263"
+PackageVersion: "6.0.15-Release.5171263"
 PackageLocale: "zh-CN"
-Publisher: "alibaba"
-PackageName: "dingtalk"
+Publisher: "Alibaba (China) Network Technology Co., Ltd."
```

```suggestion
Publisher: "Alibaba (China) Network Technology Co.,Ltd."
```
Oops sorry, there isn't supposed to be a space there actually.